README.md

```diff
@@ -15,42 +15,25 @@ Package: https://hex.pm/packages/exqlite
 
 ## Caveats
 
-* Prepared statements are not cached.
 * Prepared statements are not immutable. You must be careful when manipulating
   statements and binding values to statements. Do not try to manipulate the
   statements concurrently. Keep it isolated to one process.
-* Simultaneous writing is not supported by SQLite3 and will not be supported
-  here.
-* All native calls are run through the Dirty NIF scheduler.
-* Datetimes are stored without offsets. This is due to how SQLite3 handles date
```

Comment:
I think the Elixir -> SQLite3 mapping seems more like ecto_sqlite3 concern. `Exqlite` can focus on SQLite3 data types.